OWML.Common/Menus/IModInput.cs

```diff
@@ -3,15 +3,9 @@
 
 namespace OWML.Common.Menus
 {
-    public interface IModInput<T>
+    public interface IModInput<T> : IModInputBase
```

Comment:
This seems to be the only place that inherits from IModInputBase, so why should IModInput and IModInputBase be split?

Reply:
because IModInput<bool> != IModInput<string>

Reply:
Idea is that you can do `Container<IModInputBase>`, but can't `Container<IModInput>`